src/api.cpp

```diff
@@ -375,8 +394,8 @@ static PyObject* DataLoader_new(PyTypeObject* type, PyObject* args, PyObject* kw
             // Some kind of problem with creating the internal loader object
             std::stringstream ss;
             ss << "Unable to create internal loader object: " << e.what() << endl;
-            ERR << "Unable to create internal loader object: " << e.what() << endl;
             ss << "config is: " << json_config << endl;
+            ERR << ss.str();
             PyErr_SetString(PyExc_RuntimeError, ss.str().c_str());
             return NULL;
```

Comment:
Consider using `nullptr` instead of `NULL`

Reply:
ok